components/brave_adblock_ui/reducers/adblock_reducer.ts

```diff
@@ -1,16 +1,18 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
-/* global chrome */
 
-const types = require('../constants/adblock_types')
-const storage = require('../storage')
-let getGridSites_
+import { Reducer } from 'redux'
 
-const adblockReducer = (state, action) => {
+// Constants
+import { types } from '../constants/adblock_types'
+
+// Utils
+import * as storage from '../storage'
+
+const adblockReducer: Reducer<AdBlock.State | undefined> = (state: AdBlock.State | undefined, action) => {
```

Comment:
Redux will send you undefined state in when you combine reducers https://github.com/reduxjs/redux/blob/9b43c56b98cc75505e161b8e26c18240fa99b184/src/combineReducers.js#L68. For every reducer we have check in the beginning for undefined. I am planing of improving this, so that we would will actually set state when redux send INIT in